packages/graphback-codegen-resolvers/src/plugins/ResolverGeneratorPlugin.ts

```diff
@@ -0,0 +1,214 @@
+import { GraphbackCoreMetadata, GraphbackPlugin, ModelDefinition, getFieldName, GraphbackOperationType } from '@graphback/core';
+import { GraphQLSchema } from 'graphql';
+import { join } from 'path';
+
+export interface ResolverGeneratorPluginOptions {
+    resolverPath: string
+    // Provides extension for graphql-code-generator types
+    // generated for resolvers
+    types?: {
+        /**
+         * Name of the resolver import
+         * For example `Resolvers`
+         */
+        resolverRootType: string
+
+        /**
+         * Relative location for root resolver typings.
+         * For example: '../../types'
+         */
+        resolverRootLocation: string
+    }
+}
+
+export class ResolverGeneratorPlugin extends GraphbackPlugin {
+    private options: ResolverGeneratorPluginOptions;
+    constructor(options: ResolverGeneratorPluginOptions) {
+        super();
+
+        // TODO: default options
+        this.options = options;
+    }
+
+    public getPluginName() {
+        return 'RESOLVER_GENERATOR';
+    }
+
+    public transformSchema(metadata: GraphbackCoreMetadata): GraphQLSchema {
+        const resolvers = this.generate(metadata.getModelDefinitions());
+
+        this.write(resolvers);
+
+        return metadata.getSchema();
+        // metadata.getGraphQLTypesWithModel
+    }
+
+    private write(resolvers: { generated: any; custom: any; }) {
+        const resolversDir = join(process.cwd(), this.options.resolverPath);
```

Comment:
Process.cwd() might not be the project root. What will be your prefference:
- Pass absolute filepath
- Always pass reference against project root and enforce

`join(process.cwd(), this.options.resolverPath)` => `resolve(this.options.resolverPath)